lookout/style/format/rules.py

```diff
@@ -217,11 +217,10 @@ def fit(self, X: numpy.ndarray, y: numpy.ndarray) -> "TrainableRules":
         :param y: input labels - the same length as X.
         :return: self
         """
-        base_model = self._base_model_class(max_depth=self.max_depth,
-                                            max_features=self.max_features,
-                                            min_samples_leaf=self.min_samples_leaf,
-                                            min_samples_split=self.min_samples_split,
-                                            random_state=self.random_state)
+        base_param_names = self._base_model_class._get_param_names()
```

Comment:
Is there a public function to get the names?

Reply:
there is only a `get_params` one but it is not `@classmethod` so I can not use it here.

Reply:
Will this work?
```python
self._base_model_class().get_params()
```

Reply:
yup, I fix it and move to `base_model_name` property.